sysid/modelo.m

```diff
@@ -0,0 +1,279 @@
+function [sys,fit] = modelo(arquivo, parametros_arx, plano, aquisicao)
+
```

Comment:
Please, document the function here. You should describe its purpose and its input/output arguments. Use [ofbmdl documentation](https://github.com/lnls-dig/fofb/blob/572a5f8400df62de40aff22593b32f03b9775617/ofbmdl.m#L2-L33) as a reference.